server/src/interfaces/job.interface.ts

```diff
@@ -293,6 +298,7 @@ export enum JobStatus {
 
 export type JobHandler<T = any> = (data: T) => Promise<JobStatus>;
 export type JobItemHandler = (item: JobItem) => Promise<void>;
+export type DataTransformer = (fromExistingJob: any, fromEnqueueingJob: any) => any;
```

Comment:
Is this used anywhere?

Reply:
nope, it was in a previous impl, nuked it